app/src/main/java/com/randomappsinc/studentpicker/presentation/PresentationActivity.java

```diff
@@ -251,7 +176,7 @@ public void onPause() {
     public void finish() {
         super.finish();
         overridePendingTransition(0, R.anim.slide_out_from_top);
-        player.stop();
+        presentationManager.stopPlayer();
```

Comment:
nit: stopPlayer() made more sense when we were inlining the MediaPlayer directly. I think stopPresentation() is a better API name given the name.